src/FormBuilder/resources/js/form-builder/src/settings/design/style-controls/index.tsx

```diff
@@ -13,12 +13,14 @@ const getDesign = (designId: string) => formDesigns[designId];
  * @since 3.4.0 abstract design controls.
  */
 export default function StyleControls() {
-    const dispatch = useFormStateDispatch();
     const {
-        settings: {designId},
+        settings: {designSettingsImageUrl},
     } = useFormState();
+    const dispatch = useFormStateDispatch();
 
-    const isClassicTemplate = !getDesign(designId).isMultiStep;
+    const {
+        settings: {},
+    } = useFormState();
```

Comment:
This looks like it can be removed.

Reply:
Resolved: 0437d2cc1238c1f709ac9c9bee3b82d46efcef39